doc/rel_notes/0.7.0.txt

```diff
@@ -98,6 +98,8 @@ OpenCV Plugin
  * Added unit tests to verify that the lens distortion functions in MAP-Tk
    produce equal results to those routines in OpenCV.
 
+ * Fixed error where the size of OCV image memory was incorrectly calculated.
+
```

Comment:
Can you move this comment to the "Fixes since v0.6.0" section at the end of this document and create a new "OpenCV Plugin" sub-section there?